pwndbg/aglib/nearpc.py

```diff
@@ -168,23 +173,36 @@ def nearpc(
 
     assembly_strings = D.instructions_and_padding(instructions)
 
+    breakpoint_locations = pwndbg.dbg.breakpoint_locations()
+
     # Print out each instruction
     for i, (address_str, symbol, instr, asm) in enumerate(
         zip(addresses, symbols, instructions, assembly_strings)
     ):
         prefix_sign = pwndbg.config.nearpc_prefix
+        breakpoint_sign = pwndbg.config.nearpc_breakpoint_prefix
```

Comment:
Those two variables are re-fetched on each loop iteration. Can we move them out of the loop?